client.py

```diff
@@ -122,17 +122,24 @@ def serverFinished(self, session_id, server_nonce, client_none=None):
         if client_none is None:
             client_nonce = random.getrandbits(16*8)
         cipher = PKCS1_OAEP.new(self.server_pubkey)
+        client_nonce = client_nonce.to_bytes(16, byteorder='big')
```

Comment:
Oh, that's a lot better than what I have (`bytes(bytearray([random.getrandbits(8) for i in range(16)]))`).